README.md

```diff
@@ -9,13 +9,34 @@ The ohsome API provides various endpoints for [data aggregation](https://api.ohs
 
 ## Installation
 
-*ohsome-py* requires Python >= 3.6. The easiest way to install *ohsome-py* is using pip:
+*ohsome-py* requires
+
+* Python >= 3.6
+* geopandas >= 0.9.0
+* pyproj >= 3.0.0
+* requests >= 2.25.1
+
+### Using pip
+
+You can install *ohsome-py* using pip, which will also install all dependencies.
 
 ```
 $ pip install ohsome
 ```
 
-If you want to run the Juypter Notebook [Tutorial](https://github.com/GIScience/ohsome-py/blob/master/notebooks/Tutorial.ipynb) you also need to install `jupyter` and `matplotlib`:
+### Using Anaconda
+
+*ohsome-py* is not available through Anaconda yet. So if you are using Anaconda, create a new anaconda environment and install the required dependencies before installing *ohsome-py* using pip.
+
+```
+$ conda create -n ohsome python=3.8 geopandas requests
+$ conda activate ohsome
+$ pip install ohsome --no-deps
```

Comment:
should we add a note here? I mean anaconda and pip do not work nicely together so this is non of our business... How about we just link to the offical documentation or say `be aware that pip my seriously damage your conda environment` https://www.anaconda.com/blog/using-pip-in-a-conda-environment

Reply:
yes, I note would be good. I'll add it.